src/Combobox/index.scss

```diff
@@ -1,42 +1,10 @@
-.nds-combobox {
-  position: relative;
-}
-
```

Comment:
This was irrelevant in the first place - the dropdown is portalled outside of the nds-combobox element